lambda/src/test/resources/DigiSubs2023/readme.txt

```diff
@@ -0,0 +1 @@
+We use `regular` for the standard subscription we are going to use to test all the migration basic functions and `discounted` for the discounted subscription that we are also using to test the detection of discounted subscriptions that need to be ignored.
```

Comment:
Might be better for maintainability to put this into a readme.md at the top of the test resources as a general explanation of structure.  And then link to it from the project readme.

Reply:
Actually that explanation is only for DigiSubs2023. For that particular subscription I needed to have a discounted subscription (in addition to the standard one)